documentation/src/main/asciidoc/userguide/chapters/domain/basic_types.adoc

```diff
@@ -1829,19 +1829,40 @@ include::{sourcedir}/basic/WhereJoinTableTest.java[tags=mapping-where-join-table
 [[mapping-column-filter]]
 ==== `@Filter`
 
-The `@Filter` annotation is another way to filter out entities or collections using custom SQL criteria, for both entities and collections.
+The `@Filter` annotation is another way to filter out entities or collections using custom SQL criteria.
 Unlike the `@Where` annotation, `@Filter` allows you to parameterize the filter clause at runtime.
 
-[[mapping-filter-example]]
-.`@Filter` mapping usage
+Now, considering we have the following `Account` entity:
+
+[[mapping-filter-account-example]]
+.`@Filter` mapping entity-level usage
 ====
 [source, JAVA, indent=0]
 ----
-include::{sourcedir}/basic/FilterTest.java[tags=mapping-filter-example]
+include::{sourcedir}/basic/FilterTest.java[tags=mapping-filter-Account-example]
```

Comment:
I don't know in ORM but in other projects, we usually keep the tags all lowercase. Please ignore if it's not the case in ORM.

Reply:
I think we already used this pattern. We could address it using a separate issue.